src/Covid19.Web/Models/Countries.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Covid19Api.Models
+{
+        public class Countries
```

Comment:
i would rename this class to Country. that way when you are using a collection of Country you can use countries. Best practice is to use the singular for class names that arent collections.